defaults/cmake/Key4hepConfig.cmake

```
@@ -11,7 +12,10 @@ macro(key4hep_set_compiler_flags)
     set(COMPILER_FLAGS "${COMPILER_FLAGS} -fdiagnostics-color=always")
   endif()
 
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${COMPILER_FLAGS}")
+  if (DEFINED KEY4HEP_SET_COMPILER_FLAGS AND NOT KEY4HEP_SET_COMPILER_FLAGS)
+  else()
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${COMPILER_FLAGS}")
```

```suggestion
    set(CMAKE_CXX_FLAGS "${COMPILER_FLAGS} ${CMAKE_CXX_FLAGS}")
```
We should allow one to override the key4hep flags by package specific flags.